python/ml4ir/data/tfrecord_reader.py

```diff
@@ -99,7 +99,9 @@ def _parse_sequence_example_fn(sequence_example_proto):
                     mask = tf.expand_dims(mask, axis=2)
 
                     def crop_fn():
-                        tf.print("\n[WARN] Number of records in query : ", tf.shape(mask))
+                        tf.print(
+                            "\n[WARN] Bad query found. Number of records : ", tf.shape(mask)[1]
+                        )
                         return image.crop_to_bounding_box(
```

Comment:
we still need to use `image.crop_to_bounding_box(`?  No text-based padding yet?

Reply:
I don't think there can be - only this method allows me to pad to a **final shape**, the others expect you to specify the **extra shape you want to pad**. 
I've spent a lot of time on this previously and unsuccessfully. Can dig deeper if needed.